### DISCUSSION TITLE
Kanal İsteği

### DISCUSSION THREAD

**Discussion by gkoglanyusuf** [iptv-org/community#1084] [Q&A]
Merhaba ben uzun zamandır ip-tv kullanıyorum her kanal varda BEİN Kanalları yok. BEİN Kanallarınında eklenmesini istiyorum Türkiye Oynatma listesine.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).

**Comment by gkoglanyusuf** [comment]
merhaba İP TV Listesine Bein Kanallarınında eklenmesini talep ediyorum.

Aleksandr Statciuk ***@***.***>, 26 Şub 2025 Çar, 01:39
tarihinde şunu yazdı:
